runtime/src/lib.rs

```diff
@@ -90,8 +90,6 @@ pub enum Error {
     BlobInsufficientLength,
     #[error("offset overflow")]
     OffsetOverflow,
-    #[error("io error: {0}")]
-    Io(#[from] IoError),
```

Comment:
I suppose we never used this?

Reply:
it was used within iouring.rs in a couple of subtle places.